<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>engineering_dialog</class>
 <widget class="QDialog" name="engineering_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1515</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>engineering_dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_workflow">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>600</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="upload_button">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>440</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="extract_button">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>440</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract BOM</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>60</y>
     <width>761</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="test_button">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>440</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>1401</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Recent Uploads:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Worflow:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BOM Preview:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="table_refresh_button_2">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>30</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
